src/main/webapp/dataverse_header.xhtml

```diff
@@ -31,21 +39,35 @@
                 </div>
                 <div class="collapse navbar-collapse" id="topNavBar">
                     <ul class="nav navbar-nav navbar-right">
-                        <li class="dropdown">
-                            <a href="#" class="dropdown-toggle" data-toggle="dropdown">#{bundle['dataverse.results.btn.addData']} <b class="caret"></b></a>
-                            <ul class="dropdown-menu">
-                                <li>
-                                    <h:outputLink value="/dataverse.xhtml?ownerId=#{dataverseServiceBean.findRootDataverse().id}">
-                                        <h:outputText value="#{bundle['dataverse.results.btn.addData.newDataverse']}"/>
-                                    </h:outputLink>
-                                </li>
-                                <li>
-                                    <h:outputLink value="/dataset.xhtml?ownerId=#{dataverseServiceBean.findRootDataverse().id}">
-                                        <h:outputText value="#{bundle['dataverse.results.btn.addData.newDataset']}"/>
-                                    </h:outputLink>
+                        <ui:fragment rendered="#{showAddDataGroup}">
```

Comment:
The render logic in these duplicate `ui:fragment` components should consolidated and moved directly into the `li` components like other navbar features on this page `<li jsf:rendered="#{...}">` (line 88).

Reply:
as requested I removed the ui:fragments and moved the render logic to jsf:rendered.